protypes/Eclipse GUI with Wizards and Dialogs/src/MyPageTwo2.java

```diff
@@ -0,0 +1,100 @@
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.KeyEvent;
+import org.eclipse.swt.events.KeyListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+public class MyPageTwo2 extends WizardPage {
```

Comment:
Unfortunately, this class name does not tell me at all what the class will do. I’m additionally confused because there is neither a `MyPageOne` nor a `MyPageTwo1`.